pdf1django.txt

```diff
@@ -0,0 +1,72 @@
+Question 1;
+Model.py its file that contains python classes and each class represent db table and each attribute of the classes represent colum in the db table. 
+    in model we have models field that tells to db what the type of the colum (Boolean, Char, data, file, json).
+        model.py is take care of the db.
+
+Question 2:
+Views.py file contains function that take care of the client's requests. this function can return data from the db and stuff like that.
+    this function need to get HttpRequest object and then return Httpresponse object like html, xml and json.
+        views.py is take care of the users requests.
+
+Question 3:
+Url.py is file that contain list of urls that our Django project is support. its also can decide that each app in our project will get his own urls.py file. 
+    the list maps the url into a function that handle the client's requests. this url create bu default ones we create project. 
+        url.py take care of the url that we gonna use in the projects and send the user to the view file to the relevant function.
+
+Question 4:
+if we did any changes in our model view we need to run some commends to actually see those changes in th db. 
+    we need this to keep our db and our model synchronized /
+        to add new migrations  - mange.py migrate.
```

Comment:
Correct, but you didn't mention how to create the migrations in the first place.
It's done using `python manage.py makemigrations`